backend/layers/business/business.py

```diff
@@ -83,6 +86,32 @@ def __init__(
         self.uri_provider = uri_provider
         super().__init__()
 
+    @staticmethod
```

Comment:
nit: doesn't really need to be a static method, technically every method in this class could be, but we should stick to the established pattern